slice/Ice/Current.ice

```diff
@@ -20,7 +20,9 @@
 
 #include <Ice/ObjectAdapterF.ice>
 #include <Ice/ConnectionF.ice>
+#include <Ice/Context.ice>
```

Comment:
Both Context and OperationMode are non-local so can't be in the same file as local struct Current.